test/Microsoft.SqlTools.ServiceLayer.UnitTests/Connection/ConnectionServiceTests.cs

```diff
@@ -1219,15 +1219,5 @@ public void ConnectParamsAreInvalidIfConnectionIsNull()
 
             Assert.NotEmpty(connectionResult.ConnectionId);
         }
-
-        [Fact]
-        public async Task ConnectionWithBadConnectionStringFails()
```

Comment:
I think this is still a valid test that checks the format of a connection string, isn't it?